charts/refinery/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Refinery Helm Chart Changelog
 
+## Refinery v1.7.0
+
+- Bump app version to [v1.12.0](https://github.com/honeycombio/refinery/releases/tag/v1.12.0) (#125) | [@MikeGoldsmith](https://github.com/MikeGoldsmith)
```

Comment:
we're not super consistent about it, but in some previous chart releases we've added a short description of the underlying refinery changes corresponding to the bump

Reply:
Added a note on release changes 👍🏻